### ISSUE TITLE
Questions about the indicators of the results in the paper

### ISSUE THREAD

**Issue by ConanZH429** [possoj/Mobile-URSONet#2]
We tested it on the Speed dataset with your code, and found that the metrics in the paper could not be met on the validation set, the training set, or the entire Speed dataset, how did you get the metrics in the paper?

**Comment by possoj** [comment]
Hi @ConanZH429, the non-reproducibility issue probably comes from the non-deterministic train/validation split we used in this paper. Check out our new paper, which uses a fixed train/validation split, and you should be able to reproduce the metrics in the paper.
https://github.com/possoj/FPGA-SpacePose
